core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -153,6 +154,49 @@ function (string $value): string {
                         ->booleanNode('reject_large_uploads')
                             ->defaultValue(false)
                         ->end()
+                        ->arrayNode('sizes')
+                            ->useAttributeAsKey('name')
```

Comment:
Don't we need a configuration for the `name` attribute? And it must be validated to not be integer, otherwise it would conflict with the database.

Reply:
It is validated in `ContaoCoreExtension` class.

Reply:
I think you could also use something like this here to validate the name:

```php
->scalarNode('name')
	->validate()
	    ->ifTrue(function($name) {
	        return preg_match('/^\d+$/', (string) $name);
	    })
	    ->thenInvalid('Image size name "%s" cannot contain only digits!')
	    ->ifTrue(function($name) {
	        return in_array($name, [ResizeConfigurationInterface::MODE_BOX, ResizeConfigurationInterface::MODE_PROPORTIONAL, ResizeConfigurationInterface::MODE_CROP], true);
	    })
	    ->thenInvalid('Image size name "%s" is reserved and not allowed!')
	->end()
->end()
```

Not sure what the best practice is here though.